### DISCUSSION TITLE
bm680 error reading data

### DISCUSSION THREAD

**Discussion by Keshin1988** [pixelit-project/PixelIt#218] [Q&A] [answered]
hey guys i know its documented to use the wemos d1 mini but i use the nodemcu eps8266.
i have connected everything as the image below and so far i have everything working except the bm680
when i click dashboard it shows some values briefly but then displays error while reading.im unsure if i have done something incorrectly any help would be greatly appreciated 

![wiring](https://user-images.githubusercontent.com/114125729/193765241-6c6fadf1-79c8-4fb5-aa70-e2e39010b381.jpg)

below after saving brief display
![settings1](https://user-images.githubusercontent.com/114125729/193766413-a029112c-8b3e-497a-96ac-786c2387e961.png)

below then error
![settings](https://user-images.githubusercontent.com/114125729/193766479-7c7781d9-a795-44d1-98cf-2d2b35d5840f.png)

below my settings
![settings2](https://user-images.githubusercontent.com/114125729/193766543-e0648c12-f7ed-4197-8fe2-b2fcda9fcae6.png)

**Comment by foorschtbar** [comment]
Wiring looks good. I dont have a BME680 to test, but my two BME280 works fine. Can you try the BME680 alone? Here is a good guide: https://randomnerdtutorials.com/esp8266-nodemcu-bme680-sensor-arduino/

**Reply by Keshin1988** [reply]
thank you i did as the link you sent instructed and it is working alone

**Comment by Keshin1988** [comment, marked as answer]
ok i have resoldered to the pixelit and changed the ground from to above the 3v to bm680 and now its all working. either i had a bad solder joint or the ground fixed it. either way thank you for your help
